src/client.ts

```diff
@@ -138,6 +138,34 @@ interface ClientArgs {
   timeout?: TimeDelta;
 }
 
+interface ResultWithEvaluations {
+  id: string;
+  runId: string;
+  hash: string;
+  datasetItemId: string;
+  durationMs: number;
+  events?: Event[];
+  body: {
+    id: string;
+    name: string;
+    value: string;
+    contentType: HumanReviewFieldContentType;
+  }[];
+  output: {
+    id: string;
+    name: string;
+    value: string;
+    contentType: HumanReviewFieldContentType;
+  }[];
+  evaluations: {
+    evaluatorId: string;
+    score: number;
+    passed: boolean | null;
```

Comment:
Let's make sure what is returned from the public api reflects this
```suggestion
    passed?: boolean;
```